tests/integrationv2/test_pq_handshake.py

```diff
@@ -135,11 +136,39 @@ def assert_s2n_negotiation_parameters(s2n_results, expected_result):
         assert to_bytes(expected_result['kem_group']) in s2n_results.stdout
 
 
+def test_nothing():
+    """
+    Sometimes the pq handshake test parameters in combination with the s2n libcrypto
+    results in no test cases existing. In this case, pass a nothing test to avoid
+    marking the entire codebuild run as failed.
+    """
+    assert True
+
+
 @pytest.mark.uncollect_if(func=invalid_pq_handshake_test_parameters)
 @pytest.mark.parametrize("protocol", [Protocols.TLS12, Protocols.TLS13], ids=get_parameter_name)
+@pytest.mark.parametrize("certificate", [Certificates.RSA_4096_SHA512], ids=get_parameter_name)
 @pytest.mark.parametrize("client_cipher", CIPHERS, ids=get_parameter_name)
 @pytest.mark.parametrize("server_cipher", CIPHERS, ids=get_parameter_name)
-def test_s2nc_to_s2nd_pq_handshake(managed_process, protocol, client_cipher, server_cipher):
+@pytest.mark.parametrize("provider", [S2N], ids=get_parameter_name)
+@pytest.mark.parametrize("other_provider", [S2N], ids=get_parameter_name)
+def test_s2nc_to_s2nd_pq_handshake(managed_process, protocol, certificate, client_cipher, server_cipher, provider,
+                                   other_provider):
+    # incorrect cipher is negotiated when both ciphers are PQ_TLS_1_0_2020_12 with boringssl or libressl libcryptos
```

Comment:
It looks like openssl is included in the list of libcryptos, so maybe include that in the comment. Also, is this an issue we need to look into?
```suggestion
    # Incorrect cipher is negotiated when both ciphers are PQ_TLS_1_0_2020_12 with boringssl or libressl libcryptos
```

Reply:
Fixed: 690c3e32c0139bdfa99dd636ad4bed10a8a5d826

It could be, yes. I added a new issue for it: #3258